zebrad/src/components/mempool.rs

```diff
@@ -238,44 +246,50 @@ impl Mempool {
         is_debug_enabled
     }
 
+    /// Accepts an optional [`TipAction`] for setting a reset action's block hash as
+    /// the last seen hash, or uses the latest chain tip instead.
+    ///
     /// Update the mempool state (enabled / disabled) depending on how close to
     /// the tip is the synchronization, including side effects to state changes.
```

Comment:
Nitpick: this added comment seems like it belongs below the text that explains what the method does:
```suggestion
    /// Update the mempool state (enabled / disabled) depending on how close to
    /// the tip is the synchronization, including side effects to state changes.
    ///
    /// Accepts an optional [`TipAction`] for setting a reset action's block hash as
    /// the last seen hash, or uses the latest chain tip instead.
```

Reply:
Fixed in [`ba6f93d`](https://github.com/ZcashFoundation/zebra/pull/6416/commits/ba6f93d82eb157909ba48308eca57598c53c8c39).